devtools-ci-index.yaml

```diff
@@ -4468,15 +4468,21 @@
             ci_project: 'devtools'
             ci_cmd: '/bin/bash cico_build.sh'
             saas_git: saas-openshiftio
-            timeout: '10m'
+            timeout: '30m'
             extra_target: rhel
+        - '{ci_project}-{git_repo}-nightly':
+            git_organization: eclipse
+            git_repo: che-plugin-registry
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_build_nightly.sh'
+            timeout: '30m'
         - '{ci_project}-{git_repo}-build-master':
             git_organization: eclipse
             git_repo: che-devfile-registry
             ci_project: 'devtools'
             ci_cmd: '/bin/bash cico_build.sh'
             saas_git: saas-openshiftio
-            timeout: '10m'
+            timeout: '30m'
```

Comment:
why it is required to increase the master build timing? AFAIK, as part of it, only registry itself is built which is not time consuming